src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Runtime.props

```diff
@@ -70,6 +70,12 @@
         <TargetPath>runtimes/$(RuntimeIdentifier)/native/include/%(RecursiveDir)</TargetPath>
       </RuntimeFiles>
 
+      <RuntimeFiles Condition="'$(TargetsMacCatalyst)' == 'true' or '$(TargetsiOS)' == 'true' or '$(TargetstvOS)' == 'true'"
+        Include="@(FrameworkFiles)"
+        ExcludeFromDataFiles="true">
+        <TargetPath>runtimes/$(RuntimeIdentifier)/native/mono-framework.framework/%(RecursiveDir)</TargetPath>
```

Comment:
> mono-framework.framework

It might not be the _same_ definition of framework but still, why not just `Mono.framework` ?

Reply:
That's what it was called in Alex's WIP, I just continued with it 🤷‍♂️

Reply:
Definitely agree with Rolf, there's a reason why my change was called a WIP 😆